pwndbg/commands/diffoutput.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+
+import pwndbg.commands
+import difflib
+import pwndbg.color.message as message
+import gdb
+from pwndbg.commands.saveoutput import saved_outputs, last_command 
+
+@pwndbg.commands.Command
+def diffoutput(*args):
+    global saved_outputs, last_command
+    if args:
+        cmd = ' '.join(args)
+    else:
+        if not last_command:
+            print(message.error("No previous command to diff."))
+            return
+        cmd = last_command
+    if cmd not in saved_outputs:
+        print(message.error(f"No saved output for command: '{cmd}'"))
+        return
+        
+    try:
+        current = gdb.execute(cmd, to_string=True)
```

Comment:
Since you use gdb directly here, this command must be conditionally imported in `__init__.py` to be used only within gdb in the `if pwndbg.dbg.is_gdblib_available():` there.

Or: maybe we have some aglib way to execute cmd under any debugger?
(Note: we do support LLDB too, so it would be good if this command worked in lldb as well)


We have sth like `pwndbg.dbg.selected_inferior().evaluate_expression` but I don't see any API to run commands hm